src/google_map.js

```diff
@@ -527,6 +517,10 @@ class GoogleMap extends Component {
     />
   );
 
+  isAnotherMapInstanceMounted = () => {
+    return document.querySelector('[aria-label="Map"]');
```

Comment:
Be more specific in using the selector. A better approach would be to add a unique id in utils/mapInstance.js#initMapInstance function and look by id for that.

Reply:
Added the logic under GoogleMap util in dashboard PR